lib/Services/api.dart

```diff
@@ -0,0 +1,74 @@
+import 'dart:convert';
+import 'dart:io';
+import 'package:basic_authentication_flutter_challenge/models/person.dart';
+import 'package:http/http.dart' as http;
+
+class Authentication {
+  static String accessToken = '';
+  static String userId = '';
+  static Person person = Person();
+  static final List<Person> teamMates = [];
+
+  static Future<bool> login(String username, String password) async {
```

Comment:
I see that you are always returning a `bool` indicating whether the function has completed successfully or not. 
This is not a good practice in general. When a function completes successfully, return nothing. When it has an error, `throw` that error. The caller should have a `try ... catch` block to catch the error and display an appropriate error message.